CHANGELOG.md

```diff
@@ -1,3 +1,42 @@
+# v1.14.0
+
+## Release v1.14.0 - 2021/08/29
+
+### Enhancement
+
+* Added `enable_input_metrics`, `enable_size_metrics` system configuration parameter
+  This feature need to pay high CPU cost, so input event metrics
```

Comment:
It thinks that it's too exaggerated. Probably CPU cost isn't so increased in most cases.
```diff
- This feature need to pay high CPU cost
+ This feature might need to pay higher CPU cost
```

Reply:
Thanks, I've fixed it.